src/Microsoft.Health.Fhir.Shared.Api.UnitTests/Features/ActionResults/FhirResultTests.cs

```diff
@@ -58,5 +62,39 @@ public void GivenANotFoundStatus_WhenReturningAResult_ThenTheStatusCodeIsSetCorr
             Assert.Null(result.Result);
             Assert.Equal(HttpStatusCode.NotFound, result.StatusCode.GetValueOrDefault());
         }
+
+        [Fact]
+        public void GivenASuccessfulBundleProcessingStatus_WhenReturningAResult_ThenBadRequestIsReturned()
```

Comment:
typo : "BadRequest"

Reply:
Updated the test with based on new change